app/Http/Resources/FollowUp/FollowupVitalResources.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Resources\Vital;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class VitalResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            "id"   => $this->id,
+            "TPRBS"  => $this->TPRBS,
+            'value'  => $this->value,
+            'follow_up_id' => $this->follow_up_id,
```

Comment:
same for this as well and what does TPRBS refers to ?

Reply:
TPRBS refers to temperature, Pulse, Respiratory, BP and SP02